docs/spelling_wordlist.txt

```diff
@@ -36,6 +36,7 @@ iterable
 Koblitz
 Lange
 metadata
+multi
```

Comment:
k

On Mon, Apr 13, 2015 at 4:43 PM, Paul Kehrer <notifications@github.com>
wrote:

> In docs/spelling_wordlist.txt
> <https://github.com/pyca/cryptography/pull/1785#discussion_r28277095>:
>
> > @@ -36,6 +36,7 @@ iterable
> >  Koblitz
> >  Lange
> >  metadata
> > +multi
>
> In docker infra the "multi-threaded" line in the changelog threw a
> spelling error.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1785/files#r28277095>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084